projects/managed-hubs/incidents.md

```diff
@@ -0,0 +1,120 @@
+
+Subject Matter Expert
+  A member on the {term}`Incident Response Team` with expertise in an area of relevance to an Incident. SMEs have a variety of backgrounds and abilities, and they should be pulled in to the Response Team as-needed by the {term}`Incident Commander`. Their goal is to take actions as-directed by the {term}`Incident Commander` to resolve an incident.
+```
+
+## Communication channels
+
+### External communication
+
+- The {term}`Support Steward` team acts as the primary point of communication with external stakeholders like the {term}`Community Representative`s.
+- We may interact with external stakeholders via comments in Incident Response issues if it helps resolve the incident more quickly.
+
+### Internal communication
+
+- The Slack channel [{guilabel}`#support-freshdesk`](https://2i2c.slack.com/archives/C028WU9PFBN) contains real-time communication about support issues. Use this to signal-boost support requests related to {term}`Incident`s.
+- [Issues with the {guilabel}`incident` label](https://github.com/2i2c-org/infrastructure/issues?q=is%3Aopen+label%3A%22type%3A+Hub+Incident%22+sort%3Aupdated-desc) are where we track progress when [resolving incidents](support:incident-response).
+
+
+(support:incident-response)=
+## Incident response process
+
+Incidents are a special kind of support ticket, because they are related to degraded service that immediately impacts communities.
+We prioritize the resolution of incidents above all other kinds of work, and have a special process for tracking conversation and progress with them.
+
+Here is the process that we follow for incidents:
+
+1. **Acknowledge the incident**. Communicate with the Community Representative that there is an incident. Here is a template to get started:
+
+   ```
+   Hello { NAME }, we have investigated this request and have concluded that
+   it is related to an incident that is causing diminished service for your
+   community.
+   
+   We believe that this incident is related to { CONTEXT HERE } and will
+   investigate further on next actions. Information about our incident
+   response process can be found [in our team support documentation](https://team-compass.2i2c.org/en/latest/projects/managed-hubs/support.html).
+
+   We will open an incident report issue in [our infrastructure repository](https://github.com/2i2c-org/infrastructure)
+   where you can track progress if you wish.
+ 
+   We'll prioritize resolving this incident over our other work, and
+   will communicate with you throughout our attempts to resolve it.
+   We might be in touch with requests for clarifications if needed.
```

Comment:
Where we are asking for clarifications... I think it should be in the support Freshdesk ticket.
Then, I think we need to enforce the idea the GH issue is for internal communication and for transparency but we should avoid getting information from multiple channels.

Reply:
Have clarified this